<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Inicio</class>
 <widget class="QMainWindow" name="Inicio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>691</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="push1">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>New Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="continue_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>230</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Continue Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>260</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="loadView">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>281</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="loadBack">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="incuenta">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>170</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="loadbutton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QLabel" name="Cuenta">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>120</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Nombre de Personaje&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="invalido">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>150</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ff3806;&quot;&gt;Personaje invalido&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
